### PR TITLE
2295 copy

### LINKED ISSUE #2295: EOrust.java:183-185: Implement the "copy" method. Now it...
The puzzle `2237-a32cd01f` from #2237 has to be resolved: 

https://github.com/objectionary/eo/blob/494408a320a76b5bbfa1318815365b03020bdfca/eo-runtime/src/main/java/EOorg/EOeolang/EOrust.java#L183-L185

The puzzle was created by @rultor on 21-Jul-23. 

Estimate: 45 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #2295

---

## PR-Codex overview
This PR focuses on implementing the "copy" method in the `Universe` class. It includes the following changes:

- Added a new `copy` method in `UniverseDefault` that creates a copy of an object and returns its vertex.
- Updated the `copy` method in `Universe` interface to match the implementation in `UniverseDefault`.
- Added a new test method `copies` in `UniverseDefaultTest` to test the `copy` functionality.
- Added a new attribute "Δ" in the `DummyWithAt` class with a default value of 123L.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur could you please review these changes?

File: eo-runtime/src/test/java/org/eolang/UniverseTest.java [hunk 1]
```diff
@@ -117,6 +117,27 @@ void throwsIfWrongDataize() {
         );
     }
 
+    @Test
+    void copies() {
+        final Universe universe = new Universe(
+            new DummyWithAt(Phi.Φ)
+        );
+        final int origin = universe.find(
+            "$"
```
Comment (Graur) [inline eo-runtime/src/test/java/org/eolang/UniverseTest.java:126]: @levBagryansky Can we move `$` to the above line?

**Review by Graur — APPROVED** [review]
@levBagryansky Looks good to me
@yegor256 Please have a look

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**